service/src/main/java/com/epam/esm/services/constants/CertificateSearchParameters.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.esm.services.constants;
+
+public enum CertificateSearchParameters {
+    NAME, TAGNAME, DESCRIPTION, PRICE_LESS, PRICE_MORE;
+
+    public static CertificateSearchParameters getEntryByParameter(String s) {
```

Comment:
String s - not crear of the sense of parameter "s". What is this? naming issue.

Reply:
renamed to "parameter"